js_parser/es-simplified.esgrammar

```diff
@@ -293,12 +292,42 @@ ArgumentList[Yield, Await] :
   ArgumentList[?Yield, ?Await] `,` `...` AssignmentExpression[+In, ?Yield, ?Await]
     => arguments_append_spread($0, $3)
 
+@returns Expression
+OptionalExpression[Yield, Await] :
+  MemberExpression[?Yield, ?Await] OptionalChain[?Yield, ?Await]
+    => optional_expr($0, $1)
+  CallExpression[?Yield, ?Await] OptionalChain[?Yield, ?Await]
+    => optional_expr($0, $1)
+  OptionalExpression[?Yield, ?Await] OptionalChain[?Yield, ?Await]
+    => optional_expr($0, $1)
+
+@returns Expression
+OptionalChain[Yield, Await] :
+  `?.` `[` Expression[+In, ?Yield, ?Await] `]`
+    => optional_computed_member_expr_tail($2, $3)
```

Comment:
This is what I am not sure about... is there a better way to do this? I am not sure how to get the object from this.